NOTICE

```diff
@@ -1,5 +1,5 @@
 Apache Fluo 
-Copyright 2017 The Apache Software Foundation.
+Copyright 2017-2019 The Apache Software Foundation.
```

Comment:
However, since this is a Maven project, I believe generated files from `maven-remote-resources-plugin` will include a date range, based on the value of the `<inceptionYear>` field in the `pom.xml` and the current year. Here, however, it's sufficient to use just the current year (2019) for this static file representing the aggregate work's publication date.

Reply:
Fixed now